soc/nordic/nrf54h/CMakeLists.txt

```diff
@@ -7,6 +7,11 @@ endif()
 
 zephyr_library_sources_ifdef(CONFIG_CPU_HAS_CUSTOM_FIXED_SOC_MPU_REGIONS mpu_regions.c)
 
+if(CONFIG_PM OR CONFIG_POWEROFF)
+    zephyr_library_sources(power.c)
+endif()
```

Comment:
`power.c` only supports the ARM and not the RISCV cores - could it be excluded on non-ARM like `soc.c` above?